### ISSUE TITLE
Verify on individual resource page

### ISSUE THREAD

**Issue by MrKraken** [pwillworth/galaxyharvester#40]
Add planet selection to resource.py for when on an individual resource page for Verify click as no way to verify without going back to list page or add for planet.

Possibly make a site-wide "what planet are you on" dropdown that retains through single sessions?

**Comment by pwillworth** [comment]
Yeah, I have always felt weird about that not working.  Never thought of a solution to it that I liked before.  What do you think about instead of having a planet drop down everywhere, if someone clicks verify on a resource page, it does a pop up for them to pick a planet?

**Comment by MrKraken** [comment]
That was my original thought, also sorry I slapped issues on to start working on, I've now made an Asana board so you don't need to be pinged haha. 

From my thinking for a UX perspective. There are potential downsides obviously, but with a site-wide planet selector, it reduces the clicks needed. Make it a "Current Planet" and have a header above any tables/inputs that says "Adding Resources for: Tatooine" etc. 

I've set some time aside tomorrow to have a play around with it

**Comment by cbortz** [comment]
It may be worth consideration to allow selecting multiple planets (checkboxes or multiple-select) rather than a single option. Just my $0.02.

**Comment by MrKraken** [comment]
> 

Generally people planethop and go through the resource per planet. I think having a multi-select leads to a dangerous path, especially when you can just click a planet icon to add it to a planet, and the resource only needs to be verified on a single planet

**Comment by pwillworth** [comment]
I had another thought...  I have a feeling that people don't really use the click a planet circle to remove the spawn from that planet function.  Probably because I think a spawn goes away from all planets at pretty much the same time, so people just X the whole resource.  So, what if we just change the functionality of the planet bar such that clicking a planet circle that is already active verifies the resource on that planet, and just remove the verify link from the resource page.

This provides the functionality without adding any more UI elements.  One downside I see though is I'm not sure how intuitive it will be to people, or if we can tweak the planet bar in some way to indicate this function or make it more intuitive.

Any thoughts on this idea?  Ultimately, I'd like to avoid increasing the "form fieldiness" of Galaxy Harvester if that makes sense.

**Comment by MrKraken** [comment]
> I had another thought... I have a feeling that people don't really use the click a planet circle to remove the spawn from that planet function. Probably because I think a spawn goes away from all planets at pretty much the same time, so people just X the whole resource. So, what if we just change the functionality of the planet bar such that clicking a planet circle that is already active verifies the resource on that planet, and just remove the verify link from the resource page.
> 
> This provides the functionality without adding any more UI elements. One downside I see though is I'm not sure how intuitive it will be to people, or if we can tweak the planet bar in some way to indicate this function or make it more intuitive.
> 
> Any thoughts on this idea? Ultimately, I'd like to avoid increasing the "form fieldiness" of Galaxy Harvester if that makes sense.

This was my original thought on it, but then nixxed it because of the current behaviour. 

I still think a single planet selection may be the way to go in general though for UX (and to reduce the amount of forminess), on having a little poke about on stream, there are 6 different dropdowns for planet selection on the site that are identical but separate, then there's the planet image selection on maps.  But given the potential planet count, would probably be unwise to utilise a planet click anywhere but on the resources
